components/common/menu.tsx

```diff
@@ -19,11 +24,16 @@ function Menu() {
                     <img src="/assets/iwi-logo.svg" alt="IWI-Logo" className="h-16 md:h-24"/>
                 </a></Link>
                 <ul className="flex list-none">
-                    { items.map(item => { return menuItem(item.title, item.slug)}) }
+                    { mainMenuItems.map(item => { return menuItem(item.title, item.slug)}) }
                 </ul>
-                <Link href="/"><a className="hidden md:block">
+                <div className="dropdown hidden md:block relative">
                     <img src="/assets/user.png" alt="Zugang zum internen Bereich" className="md:h-8"/>
-                </a></Link>
+                    <div className="dropdown-menu top-0 right-0 absolute h-auto flex pt-12 hidden w-[220px]">
```

Comment:
As implemented now, the dropdown-menu's padding is blocking the access to the menu-item "Kontakt". But we need a padding between the user icon and the actual dropdown-menu to get it working. My suggested implementation would be a compromise between consistency of opening the dropdown and click-ability of the menu-item "Kontakt". 
It would also be nice, if someone would find a proper solution (if there is one)
```suggestion
                    <div className="dropdown-menu top-0 right-0 absolute h-auto flex pt-7 mt-5 hidden w-[220px]">
```